src/main/java/org/xwiki/git/internal/DefaultGitManager.java

```diff
@@ -97,13 +91,16 @@ public Repository getRepository(String repositoryURI, String localDirectoryName,
 
             // Step 2: Verify if the directory exists and isn't empty.
             if (!gitDirectory.exists()) {
-                CloneCommand cloneCommand = git.cloneRepository();
+                CloneCommand cloneCommand = Git.cloneRepository();
+                // Bare becomes true if useBare is received true, otherwise false.
```

Comment:
I don't find that comment very useful, do you? :)

Reply:
Nope. Let me remove it.